src/ssl.c

```diff
@@ -45557,6 +45557,9 @@ int wolfSSL_X509_NAME_print_ex(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
             return WOLFSSL_FAILURE;
     }
 
+    if (name->sz == 0)
```

Comment:
Further down sz is used with -1 and -2.

Reply:
Added more checks in appropriate conditions

Reply:
This API `wolfSSL_X509_NAME_print_ex` is public and we are not checking `name` for NULL... Please add.

Reply:
Check